tests/drivers/disk/disk_access/src/main.c

```diff
@@ -147,7 +150,11 @@ static void test_sector_write(uint8_t *wbuf, uint8_t *rbuf, uint32_t num_sectors
 	rc = write_sector_checked(wbuf, rbuf, 0, num_sectors);
 	zassert_equal(rc, 0, "Failed to write to sector zero");
 	/* Write to a sector in the "middle" of the disk */
-	sector = MAX(((disk_sector_count / 2) - num_sectors), 0);
+	if ( disk_sector_count / 2 > num_sectors )
+                sector = disk_sector_count / 2 - num_sectors;
+        else
+                sector = 0;
```

Comment:
all if/elses need `{}`, also please don't mix tabs and spaces...

Reply:
Yes, CI found that problem.
I have updated the format. 
Thanks